.github/workflows/subctl_bleeding_edge.yml

```
@@ -15,7 +15,11 @@ jobs:
       - name: Generate the Artifacts
         run: |
           make build-cross
```

```suggestion
          make build-cross VERSION=devel
```

I just realized that since it's in the Makefile now, you can just do this :D